src/pages/DepositBTC.tsx

```diff
@@ -161,26 +160,24 @@ function DepositBTC(): ReactElement | null {
   function updateTokenFormValue(symbol: string, value: string): void {
     updateTokenFormState({ [symbol]: value })
   }
-
   const depositData = {
-    shareOfPool: parseFloat(
-      formatUnits(
-        poolData?.totalLocked.gt(0)
-          ? estDepositLPTokenAmount
-              .mul(BigNumber.from(10).pow(18))
-              .div(poolData?.totalLocked)
-          : BigNumber.from(0),
-        18,
-      ),
-    ).toFixed(2),
-    lpToken: commify(
-      parseFloat(formatUnits(estDepositLPTokenAmount, 18)).toFixed(5),
+    shareOfPool: formatBNToPercentString(
+      poolData?.totalLocked.gt(0)
```

Comment:
@alphastorm new math for share of pool

Reply:
<img width="477" alt="Screen Shot 2021-01-27 at 5 50 47 PM" src="https://user-images.githubusercontent.com/7607886/106078403-35f66d00-60c8-11eb-8fe2-be29525664fc.png">